tests/integrationTests/inputs/compressibleCouetteFlow.yaml

```diff
@@ -71,16 +71,29 @@ solver: !ablate::finiteVolume::CompressibleFlowSolver
         fieldName: euler
         field: "1.0, 216500.0, 50.0, 0.0"
   monitors:
+    - !ablate::monitors::SolutionErrorMonitor
```

Comment:
Did you update the outputs file to account for these changes?

Reply:
I have run the integration test yaml and the tests passed locally. I am not sure what you meant by updating the output file.

Reply:
Can you try running them again?  When I checkout your code and run the integration tests they fail.  You will need to update the corresponding expected output in `tests/integrationTests/outputs/compressibleCouetteFlow.txt`

`
/Users/mcgurn/scratch/ablate/tests/testingResources/MpiTestFixture.cpp:150: Failure
Expected equality of these values:
  expectedLine
    Which is: "Timestep: 0000 time = 0        \t l2_norm error:"
  actualLine
    Which is: "Timestep: 0000 time = 0        \t l1_norm error:"
 on line Timestep: 0000 time = 0        	 l2_norm error: of file outputs/compressibleCouetteFlow.txt
Generated output file: Tests_IntegrationTestsSpecifier.ShouldRun_inputs_compressibleCouetteFlow_yaml.txt
`

Reply:
I see what you mean. I have changed the output .txt to match what I see in the printout outputs. I remember that I can run individual test from the Clion UI. How can I do that? Right now I am running All CTest.